test/infrastructure/clusters/test-infra/karpenter-tests/scripts/cleanup.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
+# Clean up the CI account from failed CF stacks or failed test runs
+# The script assumes the aws CLI command is authenticated AWS_REGION is set
+
+# EXPIRATION_TIME is the exact time in UTC that resources expire and will be swept by the script
+EXPIRATION_TIME=$(date -d 'now-12 hours' +%Y-%m-%dT%H:%M --utc)
+echo "Expiration time for all resources is: $EXPIRATION_TIME"
+
+# Grab all instances that are older than the EXPIRATION_TIME and then filter out the KITInfrastructure instances
+old_instances=($(aws ec2 describe-instances --filters Name=instance-state-name,Values=running --query "Reservations[].Instances[?LaunchTime <= '$EXPIRATION_TIME'][]" | grep -v "kubernetes.io/cluster/KITInfrastructure" | jq -r '.[].InstanceId' | sort))
+n_old_instances="${#old_instances[@]}"
+echo "Removing ${n_old_instances} old instances"
+
+if (( n_old_instances > 0 )); then
+  aws ec2 terminate-instances --instance-ids ${old_instances}
```

Comment:
Any thought on doing the following so one failure doesn't stop the rest of the script? 
```suggestion
  aws ec2 terminate-instances --instance-ids ${old_instances} || true
```